cvat/apps/iam/tests/test_rest_api.py

```diff
@@ -20,6 +29,21 @@
          name='account_email_verification_sent'),
 ]
 
+class ForceLogin:
+    def __init__(self, user, client):
+        self.user = user
+        self.client = client
+
+    def __enter__(self):
+        if self.user:
+            self.client.force_login(self.user, backend='django.contrib.auth.backends.ModelBackend')
```

Comment:
Probably here we can configure huge limits for the user. What do you think? Let's add a function that writes big values for the user.